src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -109,6 +109,13 @@ export default class Result extends React.Component {
     return <Note>{t(`Displaying up to ${NUMBER_OF_SERIES_BY_DAY} results`)}</Note>;
   }
 
+  truncate(string) {
+    if (string.length > 80) {
+      return string.substring(0, 80);
```

Comment:
You could skip the `if` check since substring already handles the case where length < endIndex. It'll allow you to avoid hardcoding 80 on 2 lines.